docs/user_guides/fs/feature_view/model-dependent-transformations.md

```diff
@@ -0,0 +1,131 @@
+            ]
+        )
+        ```
+
+### Using built-in transformations
+
+Built-in transformation functions are attached in the same way. The only difference is that they can either be retrieved from the Hopsworks or imported from the `hsfs` module.
+
+=== "Python"
+
+    !!! example "Creating model-dependent transformation using built-in transformation functions retrieved from Hopsworks"
+        ```python
+        min_max_scaler = fs.get_transformation_function(name="min_max_scaler")
+        standard_scaler = fs.get_transformation_function(name="standard_scaler")
+        robust_scaler = fs.get_transformation_function(name="robust_scaler")
+        label_encoder = fs.get_transformation_function(name="label_encoder")
+        
+        feature_view = fs.create_feature_view(
+            name='transactions_view',
+            query=query,
+            labels=["fraud_label"],
+            transformation_functions = [
+                label_encoder("category"),
+                robust_scaler("amount"),
+                min_max_scaler("loc_delta"),
+                standard_scaler("age_at_transaction")
+            ]
+        )
+        ```
+
+To attach built-in transformation functions from the `hsfs` module they can be directly imported into the code from `hsfs.builtin_transformations`.
+
+=== "Python"
+
+    !!! example "Creating model-dependent transformation using built-in transformation functions imported from hsfs"
+        ```python
+        from hsfs.builtin_transformations import min_max_scaler, label_encoder, robust_scaler, standard_scaler
+        
+        feature_view = fs.create_feature_view(
+            name='transactions_view',
+            query=query,
+            labels=["fraud_label"],
+            transformation_functions = [
+                label_encoder("category": ),
+                robust_scaler("amount"),
+                min_max_scaler("loc_delta"),
+                standard_scaler("age_at_transaction")
+            ]
+        )
+        ```
+
+
+## Using Model Dependent Transformations
+
+Model-dependent transformations attached to a feature view are automatically applied when you [create training data ](./training-data.md#creation), [read training data](./training-data.md#read-training-data), [read batch data](./batch-data.md#creation-with-transformation), or [get feature vectors](./feature-vectors.md#retrieval-with-transformation). The generated data includes untransformed features, on-demand features, if any, and the transformed features. The transformed features are organized by their output column names in alphabetical order and are positioned after the untransformed and on-demand features. 
+
+Model-dependent transformation functions can also be manually applied to a feature vector using the `transform` function. 
+
+=== "Python"
+
+    !!! example "Manually applying model-dependent transformations during online inference"
+        ```python
+        # Initialize serving 
```

Comment:
Initialize the feature view with the correct training dataset version used for model-dependent transformations